rest_docs/rest-catalog-open-api.yaml

```diff
@@ -362,11 +362,142 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+    post:
+      tags:
+        - Catalog API
+      summary: Create a table in the given namespace
+      description: Create table
+      operationId: createTable
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateTableRequest'
+      responses:
+        200:
+          $ref: '#/components/responses/CreateTableResponse'
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        401:
+          $ref: '#/components/responses/UnauthorizedResponse'
+        404:
+          description: Not Found - The namespace specified does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/IcebergErrorResponse'
+              examples:
+                NamespaceNotFound:
+                  $ref: '#/components/examples/NoSuchNamespaceError'
+        5XX:
+          $ref: '#/components/responses/ServerErrorResponse'
+
+  /v1/namespaces/{namespace}/tables:
```

Comment:
Possible bug: I think this line is intended to be above the `post` section above? it seems off to have two routes, one after another.

Reply:
Fixed.